std/meta.d

```diff
@@ -1309,6 +1309,52 @@ private template staticMerge(alias cmp, int half, Seq...)
     }
 }
 
+/**
+ * Allows subsetting of multiple members of tuple types that are not
+ * adjacent to each other, i.e. cannot be sliced.
+ *
+ * Evaluates to $(D AliasSeq!(T[range[0]], T[range[1]], ..., T[range[n]])).
+ */
+
+template staticSubset(alias range, T...)
```

Comment:
The documentation is exceedingly thin. What happens when `range` has more elements than `T`? Does `range` have to be a random access range as could reasonably be inferred from the summary and example? I know the answer to these questions only because I've seen the implementation. They should be reflected in documentation.